### PR TITLE
Deduplicate some code

### PR DESCRIPTION
**Pull request by mvdan**

### CODE REVIEW

File: rpc_storage_handler.go [hunk 1]
```diff
@@ -596,31 +596,8 @@ func (r *RPCStorageHandler) SetRollingWindow(keyName string, per int64, val stri
 
 }
 
-// SetScrollingWindow is used in the rate limiter to handle rate limits fairly.
 func (r *RPCStorageHandler) SetRollingWindowPipeline(keyName string, per int64, val string) (int, []interface{}) {
```
Comment (buger) [inline rpc_storage_handler.go]: So, if this function do not have any differences from `SetRollingWindow`, maybe make sense to remove it a all, and replace all references by `SetRollingWindow` ?

Comment (mvdan) [inline rpc_storage_handler.go]: I tried this first, but it doesn't work at all. I forgot to mention why in the commit, will update.

Comment (mvdan) [inline rpc_storage_handler.go]: @buger done, see the new commit message

Comment (buger) [inline rpc_storage_handler.go]: Got it, make sense

**Pull request merged**